code/src/Question3.js

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+
+const Question3 = ({ training, onTrainingChange }) => {
+  return (
+    <>
+      <section>
+        <div className="question-3">
+          <div>
+            <label htmlFor="trainingKickboxing">Kickboxing</label>
+            <input
+              id="trainingKickboxing"
+              type="radio"
+              checked={training.includes("Kickboxing")}
```

Comment:
This is how you would deal with checkboxes, not radio buttons :) Radio buttons work like only one at a time should be selected.